### ISSUE TITLE
[Harvester] when a cluster member is added and then removed from the harvester cluster RBAC logged in as a Cluster owner all the members disappear

### ISSUE THREAD

**Issue by anupama2501** [rancher/dashboard#4347]
- Create a local user from rancher --> Users and auth --> create user1,user2 [standard user permissions]
- As an admin from virtualization management --> harvester cluster --> RBAC --> add the user1 as cluster Owner
- Login as user1 and verify the harvester cluster is seen
- As user1 [Cluster owner] virtualization management --> harvester cluster --> RBAC --> add user2 and user1 as a cluster member
- Remove user1 cluster member logged in as user1 from the RBAC page 
- All the members from the page disappear and a 404 response is seen in the network logs:

```
xhr.js:178 GET https://<redacted>/v1/management.cattle.io.clusterroletemplatebindings/c-m-fqrjnx4k/crtb-4d5ds 404 (Not Found)
(anonymous) @ xhr.js:178

``` 

Note: 

- Refresh the UI and the users are seen listed in the RBAC
- Create a new user and verify all the users are listed. 
- The issue happens sometimes. 

ref: Found when verifying the ticket https://github.com/rancher/dashboard/issues/4299

**Comment by richard-cox** [comment]
@n313893254 I noticed you owned the matching harvester issue (https://github.com/harvester/harvester/issues/1377). When the fix this is merged to `rancher/dashboard` `master` please can you let me know and I'll move this issue to the required state

**Comment by TachunLin** [comment]
Verified **fixed** on master-f5bde7fa-head with Rancher `v2.6.3-rc1`.
Please comment whether we can close this issue it's related to [harvester/harvester/#1377](https://github.com/harvester/harvester/issues/1377) or the followup steps. 

### Result
In harvester cluster member (RBAC) page, login user1 (owner) -> add user2 (member) -> add user1 (member) -> delete user1(member)  then navigate to other page in harvester then login RBAC again. 

All the members are displayed in the cluster member page correctly without blank page. 
![image](https://user-images.githubusercontent.com/29251855/143068643-c035b013-3aa4-4675-b63b-73c711f7828b.png)

Logout and login user1 again, it still can 

### Test Information
* Test Environment: 3 nodes harvester on provo bare machines
* Harvester version: master-f5bde7fa-head
* Rancher version: v2.6.3-rc1

### Environment Preparation
1. Install harvester v2.6.3-rc1 through docker command

### Verify Steps

1. Environment preparation as above steps
2. Import harvester in rancher 
3. Create `user1` and `user2` as standard user in user&authentication
![image](https://user-images.githubusercontent.com/29251855/143063544-d72412d2-4bcd-46a0-b89a-06c6ec7a1a66.png)
4. Add user1 as cluster owner in harvester RBAC page
5. Login with user user1 and access harvester RBAC page
6. Add user2 as cluster member 
7. Add user1 as cluster member
![image](https://user-images.githubusercontent.com/29251855/143066665-6aef3410-faeb-4973-9aef-4cb8c38df534.png)
8. Delete user1 (cluster member)
![image](https://user-images.githubusercontent.com/29251855/143064949-12ede9dd-00b7-44a2-b815-ab6ff6ca969f.png)
9. Navigate to other page (e.g image)
10. Access RBAC page again 
11. Logout and login with user1 again 
12. Access RBAC page

**Comment by guangbochen** [comment]
close as it passes the above validation.
